reana/reana_dev/run.py

```diff
@@ -297,6 +374,9 @@ def run_example(
     :param workflow_engine: The option ``workflow_engine`` can be repeated. The
                             value is the workflow engine to use to run the
                             example. [default=cwl,serial,yadage]
+    :param compute_backend: The option ``compute_backend`` can be repeated. The
+                            value is the compute backend to use to run the
+                            example. [default=kubernetes,htcondorcern,slurmcern]
```

Comment:
The help strings could be a bit misleading, since people might be tempted to write `-b htcondorcern,slurmcern` while the command expects multiple `-b` arguments.  Ditto for worklow engines or demos to run.  Example:

```console
$ reana-dev run-example -c r-d-helloworld -c r-d-worldpopulation -c r-d-r-roofit -b ALL -w serial -w yadage --submit-only
```

Perhaps we can amend either the cosmetics, or accept comma-separated lists,  like for `--exclude-components` ?